server/routes/varyingLicences/handlers/caseload.ts

```diff
@@ -20,9 +20,10 @@ export default class CaseloadRoutes {
         ? await this.comCaseloadService.getTeamVaryCaseload(user, req.session.teamSelection)
         : await this.comCaseloadService.getStaffVaryCaseload(user)
     ).map(comCase => {
+      const releaseDate = comCase.releaseDate ? format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy') : 'Not Found'
```

Comment:
*nitpick: * Elsewhere in the app we use lowercase of 'not found'
If this ends up being a short-term fix, I think it's fine to go out, but if it hangs around more than a couple of days, UCD may want us to correct it

Reply:
ok I may as well fix it now

Reply:
Fixed, can you approve when it's built again, thanks